backend/tsconfig.json

```diff
@@ -8,6 +8,17 @@
       "strict": true,
       "forceConsistentCasingInFileNames": true,
       "outDir": "dist",
-      "moduleResolution": "node"     
+      "moduleResolution": "node",
+      "baseUrl": "src",
+      "paths": {
+        "src/*": [
+          "./src/*"
+        ]
+      }     
+    },
+    "include": ["./src/**/*.ts"],
+    "exclude": ["node_modules", "dist"],
+    "ts-node": {
+      "require": ["tsconfig-paths/register"]
```

Comment:
What is the purpose? Please change it according to the tsconfig.json on develop branch

Reply:
I dont fix this file - its merge from develop branch before